### DISCUSSION TITLE
Is this preferred over Feedback Hub?

### DISCUSSION THREAD

**Discussion by riverar** [microsoft/WSA#53] [General]
It's not clear if this repository is favored over Feedback Hub or generally where FBH stands in terms of the team's triage priority.

**Comment by hamza-usmani** [comment]
Hi, this repository is not for end-user related issues for apps/the Amazon Appstore. This repository is specifically to target developer and developer-related issues with the subsystem! Windows Feedback Hub should be for end-user related issues.
